### PR TITLE
Use ArchiveService from RDW_Common.

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/ImportServiceApplication.java [hunk 1]
```diff
@@ -16,10 +17,14 @@
 		ContextStackAutoConfiguration.class
 })
 @PropertySource(value = "classpath:build-info.properties", ignoreResourceNotFound = true)
-@Import({YamlPropertiesConfigurator.class, StatusConfiguration.class})
+@Import({
+		YamlPropertiesConfigurator.class,
```
Comment (malaffoon) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/ImportServiceApplication.java]: hmm, i'll check the whitespace on this

File: import-service/src/test/java/org/opentestsystem/rdw/ingest/status/DatabaseStatusIndicatorTest.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public void createStatusIndicator() {
         });
 
         archiveService = mock(ArchiveService.class);
-        when(archiveService.readResource("status")).thenReturn("status check");
+        when(archiveService.readResource("status")).thenReturn("status check".getBytes());
```
Comment (malaffoon) [inline import-service/src/test/java/org/opentestsystem/rdw/ingest/status/DatabaseStatusIndicatorTest.java]: in a test like this the encoding doesn't matter

File: build.gradle [hunk 3]
```diff
@@ -42,6 +42,7 @@ subprojects {
 
     dependencyManagement {
         dependencies {
+            dependency 'org.opentestsystem.rdw.common:rdw-common-archive:0.0.1-SNAPSHOT'
```
Comment (rcordeau) [inline build.gradle]: When we move to appending build numbers to the versions, do you want to remove SNAPSHOT?

Comment (malaffoon) [inline build.gradle]: Absolutely.

**Pull request merged**